contracts/connection-router/src/types.rs

```diff
@@ -78,6 +81,66 @@ impl KeyDeserialize for MessageID {
     }
 }
 
+/// cosmwasm cannot serialize tuples, so we need to convert [GlobalMessageId] into a struct
+#[cw_serde]
+struct GlobalMessageIdSerde {
+    chain_name: ChainName,
+    message_id: MessageID,
+}
+
+#[cw_serde]
+#[serde(from = "GlobalMessageIdSerde", into = "GlobalMessageIdSerde")]
+pub struct GlobalMessageId(ChainName, MessageID);
```

Comment:
Why not define these types in state.rs?

Reply:
I think I defined the GlobalMessageId where the MessageID was defined, so that happened to be `types.rs`. I'd like to get rid of this catchall file entirely, but didn't want to put that in the same PR